db/create-database.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
If you want to isolate scripts per docker, can you please move `db` folder to `infra/docker`?